scoap3/utils/create_goups.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth.models import Group, Permission
```

Comment:
maybe we can use something like this for these default groups/permissions, so we have less extra code https://github.com/SCOAP3/scoap3/commit/6d663c6675e56994b629551af80a31975c9b4890#diff-fdaec76e8797f00d4352cf11042e90c117b7de84c426e58275a8d3154dbde2f8

After we will be able to do smthg like:
`python manage.py loaddata default_groups` and have the same result

https://docs.djangoproject.com/en/4.2/howto/initial-data/#provide-data-with-fixtures